Games/Warcraft III TFT/Online/script.json

```diff
@@ -0,0 +1,7 @@
+{
+	"scriptName": "Online",
+	"compatibleOperatingSystems": ["LINUX"],
```

Comment:
You should not set "LINUX" for `compatibleOperatingSystems` and `testingOperatingSystems`. If it works, use `compatibleOperatingSystems`, otherwise `testingOperatingSystems`.